lib/OpenQA/Worker/Jobs.pm

```diff
@@ -400,7 +400,7 @@ sub _stop_job_2 {
 
 sub _stop_job_finish {
     my ($params, $quit) = @_;
-    log_debug("update status running $update_status_running") if $verbose;
+    log_debug("update status running $update_status_running") if $verbose && $update_status_running;
```

Comment:
if we only output this line if it's set, we don't need the value in the string IMO. it's boolean